src/components/Mentor/Mentor.jsx

```diff
@@ -0,0 +1,117 @@
+import React, { useState, useEffect } from "react";
+import { Row, Col } from "reactstrap";
+import defaultprofile from "../../../static/logos/profile-placeholder.png";
+import moment from "moment-timezone";
+import MentorModal from "../MentorModal/MentorModal.jsx";
+
+const Mentor = () => {
+  const [mentors, setmentors] = useState([]);
+  const getmentor = async () => {
+    const url =
+      "https://api.trello.com/1/lists/5eb715b48caa18614425c25e/cards?fields=name,labels,cover,desc&customFields=true&customFieldItems=true&attachments=true&attachment_fields=all";
+
+    const response = await fetch(url);
```

Comment:
Use `try-catch` blocks when using await statements instead of `if-else`.

Reply:
I use If-else because API doesn't work so we can display an Alert same as we are displaying on Download Page, just pushed the changes please have a look

Reply:
you could just keep it as async

https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API/Using_Fetch

Reply:
I think it's the same thing just a bit different syntax we are using JS ES6 and on the doc, it's an older version. If you are talking about directly fetching the information from API , In the future, it might not work and so we need async and await to check response is coming or not